### PR TITLE
bug_report improvements

### PR DESCRIPTION
**Pull request by ilyam8**
##### Summary

Fixes: #7656

##### Component Name

`.github/ISSUE_TEMPLATE`

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -4,31 +4,52 @@ about: Create a bug report to help us improve
...
+##### OS / Environment
+<!--
+Provide as much information about your environment (OS distribution, running in container, etc.)
+as possible to allow us reproduce this bug faster.
 
-##### Netdata version (ouput of `netdata -V`)
+To get this information execute:
+- uname -a; grep -Hv "^#" /etc/*release  # linux/bsd
+- uname -a; sw_vers                      # macOS
+
```
Comment (thiagoftsm) [inline .github/ISSUE_TEMPLATE/bug_report.md]: The previous 3 lines should be on `OS /Environment`

Comment (thiagoftsm) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Sorry, I only saw now that you are removing Netdata version.

Comment (ilyam8) [reply .github/ISSUE_TEMPLATE/bug_report.md]: i havent remove anything, i moved stuff to appropriate section.

> The previous 3 lines should be on OS /Environment

Those limes under `OS /Environment` section.

**Comment by ilyam8** [comment]
@thiagoftsm 
https://raw.githubusercontent.com/ilyam8/netdata/bug_template_improvements/.github/ISSUE_TEMPLATE/bug_report.md

**Comment by ilyam8** [comment]
btw

> uname -a; grep -Hv "^#" /etc/*release  # linux/bsd

is not correct, it recall it doesnt work on (at least all) bsd systems, should be improved.

**Comment by amoss** [comment]
> btw
> 
> > uname -a; grep -Hv "^#" /etc/*release  # linux/bsd
> 
> is not correct, i recall it doesnt work on (at least all) bsd systems, should be improved.

This has just been updated in `daemon/system-info.sh`, there is an example in there on how to find this for FreeBSD.

**Comment by ilyam8** [comment]
https://github.com/netdata/netdata/blob/49f6950820f083956ddc2c9327eeae4e9ef0d5ed/daemon/system-info.sh#L95-L101

This?

so just `uname -a` ?

**Comment by amoss** [comment]
Adding the `uname -K` gives the explicit kernel version, but it is the other part that does not work on all BSDs isn't it? There is no `/etc/release` on FreeBSD for example.

What about something like:
```
sw_vers 2>/dev/null || grep -Hv "^#" /etc/*release || (uname -a; uname -K)
```

**Comment by ilyam8** [comment]
> sw_vers 2>/dev/null || grep -Hv "^#" /etc/*release 2>/dev/nul || (uname -a; uname -K)


i dont see reason why we need to have one line (which is big already and will grow in the future 100%) if we can have 3:

- linux
- bsd
- macos

It is even easier for me - clear what is what.

**Comment by joelhans** [comment]
I'll try to review this soon. Does the SRE team know about this yet? I feel like one of them said something at the meetup about optimizing this very template. Maybe drop in their channel to see if anyone wants to participate.

**Comment by Ferroin** [comment]
I second @prologic's assessment, but will hold off officially reviewing as this should also be approved by @joelhans.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -4,31 +4,53 @@ about: Create a bug report to help us improve
...
+Provide as much information about your environment (OS distribution, running in container, etc.)
+as possible to allow us reproduce this bug faster.
 
-##### Netdata version (ouput of `netdata -V`)
+To get this information execute:
+- uname -a; grep -Hv "^#" /etc/*release  # linux
+- uname -a; uname -K                     # bsd
+- uname -a; sw_vers                      # macOS
+
+Place output in the code section.
```
Comment (joelhans) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
Place the output from the command in the code section below.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -4,31 +4,53 @@ about: Create a bug report to help us improve
...
+ -->
+```
+
+```
+
+##### Netdata version
+<!--
+Provide output of netdata -V.
+ 
+If netdata is running execute: $(ps aux | grep -E -o "[a-zA-Z/]+netdata ") -V
```
Comment (joelhans) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
If Netdata is running, execute: $(ps aux | grep -E -o "[a-zA-Z/]+netdata ") -V
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -4,31 +4,53 @@ about: Create a bug report to help us improve
...
+ -->
+ 
 
 ##### Component Name
+<!--
+Write which component is affected. We group our components the same way our code is structured so basically: 
+component name = dir in top level directory of repository.
+-->
```
Comment (joelhans) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Let's try something like this:

Let us know which component is affected by the bug. Our code is structured according to its component, so the component name is the same as the top level directory of the repository. For example, a bug in the dashboard would be under the web component.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -4,31 +4,53 @@ about: Create a bug report to help us improve
...
 
 ##### Steps To Reproduce
+<!--
+Describe how you found this bug and how we can reproduce it. Preferable with a minimal test-case scenario.
```
Comment (joelhans) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
Describe how you found this bug and how we can reproduce it, preferably with a minimal test-case scenario.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -4,31 +4,53 @@ about: Create a bug report to help us improve
...
+You can paste gist.github.com links for larger files.
```
Comment (joelhans) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
If you'd like to attach larger files, use gist.github.com and paste in links.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 7]
```diff
@@ -4,31 +4,53 @@ about: Create a bug report to help us improve
...
+##### OS / Environment
+<!--
+Provide as much information about your environment (OS distribution, running in container, etc.)
+as possible to allow us reproduce this bug faster.
 
-##### Netdata version (ouput of `netdata -V`)
+To get this information execute:
+- uname -a; grep -Hv "^#" /etc/*release  # linux
+- uname -a; uname -K                     # bsd
+- uname -a; sw_vers                      # macOS
```
[the remark this answers is not in the document]
Comment (Ferroin) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I _think_ that should work on all modern BSD variants, but we technically only support FreeBSD right now.

**Pull request merged**